test/api/extensions/customer-ext.spec.js

```diff
@@ -1,56 +1,56 @@
-const supertest = require("supertest")
-const app = require("../../../src/server.js")
-const { createCustomer } = require("../../helpers/createCustomer.js")
+const supertest = require("supertest");
```

Comment:
Just need to write the extra extension tests to make sure you're handling all the queries correctly, looks all good otherwise.